cmd/build/helmify/kustomize-for-helm.yaml

```diff
@@ -76,7 +76,7 @@ spec:
             - --logtostderr
             - --log-denies={{ .Values.logDenies }}
             - --emit-admission-events={{ .Values.emitAdmissionEvents }}
-            - --log-level={{ .Values.logLevel }}
+            - --log-level={{ (.Values.controllerManager.logLevel | empty | not) | ternary .Values.controllerManager.logLevel .Values.logLevel }}
```

Comment:
do we need to add these to https://github.com/open-policy-agent/gatekeeper/blob/master/cmd/build/helmify/static/values.yaml

Reply:
We can if you want, but from the helm perspective, I don't think it's necessary.

Reply:
nit: Would be nice to at least add it to the helm docs here: https://github.com/open-policy-agent/gatekeeper/blob/00bc24f6c5d67b4fa3248f6197dc9726a9436771/cmd/build/helmify/static/README.md

Reply:
Added: 938c308e3aeff4327a4ebb8d83b4e5715051de91

Reply:
@congiv can you pls run `make manifests` again to get the README changes into manifest_staging? Thanks!

Reply:
done